pkg/virt-handler/vm.go

```diff
@@ -2644,8 +2644,12 @@ func (d *VirtualMachineController) finalizeMigration(vmi *v1.VirtualMachineInsta
 	if err != nil {
 		return err
 	}
-	err = client.FinalizeVirtualMachineMigration(vmi)
-	if err != nil {
+
+	if err := isolation.AdjustQemuProcessMemoryLimits(d.podIsolationDetector, vmi); err != nil {
+		log.Log.Object(vmi).Reason(err).Warning("failed to adjust QEMU process memory limits")
```

Comment:
I think that the text should be `"failed to finalize migration"` and the `err` to contain the details (which I am pretty sure already exists).

Reply:
Done,
Also changed the error messages of the related functions, no it will return proper error message with no duplication.